cove_ocds/management/commands/ocds_cli.py

```diff
@@ -30,10 +31,28 @@ def handle(self, file, *args, **options):
             ))
 
         try:
-            result = ocds_json_output(self.output_dir, file, schema_version, convert)
+            if os.path.isdir(file):
```

Comment:
The behavior between the file and directory are not the same. A file produces a full `results.json` file, but a directory produces a CSV with only the validation errors.

I would recommend having a directory produce a JSON Lines file (one line of JSON for each file). An external script can read that the JSON Lines file to produce a CSV if that's what's desired.